core/receiver/receiver.go

```diff
@@ -9,7 +9,7 @@ import (
 type Repository interface {
 	List(context.Context, Filter) ([]Receiver, error)
 	Create(context.Context, *Receiver) error
-	Get(context.Context, uint64) (*Receiver, error)
+	Get(context.Context, uint64, bool) (*Receiver, error)
```

Comment:
we don't need to keep the old behaviour

Reply:
Didn't got this can you explain this further

Reply:
Let me start with question. Probably I understand it wrong.

Why we need to add bool here?

Reply:
With this bool here we are defining weather we want to expand the get call or not.

Reply:
In that case, lets use filter like the one being used by List?